Bottom: 106a7e840a912ca03996e883b013627e287f24a8
Top:    8f4dd14fef8c75965652fe27e1913242bdbc773c
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2014-08-10 22:02:03 -0400

Refresh of tyler

---

diff --git a/sound/soc/generic/simple-card.c b/sound/soc/generic/simple-card.c
index 7b6edb6..8840c36 100644
--- a/sound/soc/generic/simple-card.c
+++ b/sound/soc/generic/simple-card.c
@@ -147,7 +147,6 @@ asoc_simple_card_sub_parse_of(struct device_node *np,
 	 */
 	if (of_property_read_bool(np, "clocks")) {
 		clk = of_clk_get(np, 0);
-printk("JDS clk %p\n", clk);
 		if (IS_ERR(clk)) {
 			ret = PTR_ERR(clk);
 			return ret;
